assignment09/part01/index.php

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 1</title>
+</head>
```

Comment:
This index file should be a form taking in values for height and weight from the user. There was starter code in the zip file.